invenio_app/factory.py

```diff
@@ -59,30 +58,13 @@ def static_url_path():
 
 
 def config_loader(app, **kwargs_config):
-    """Configuration loader.
-
-    Adds support for loading templates from the Flask application's instance
-    folder (``<instance_folder>/templates``).
-    """
-    # This is the only place customize the Flask application right after
```

Comment:
Yes, if used in combination with https://github.com/inveniosoftware/invenio-base/pull/151

Reply:
Fun-fact: the bytecache option was wrongly initialized, since it was called before the `invenio_config_loader` call and thus `CACHE_KEY_PREFIX` couldn't be set e.g. via `invenio.cfg`. Actually even now it's still an issue, since the `invenio-cache` extension is still not initialized, so no `CACHE_*` config is loaded...